x/staking/keeper/delegation.go

```diff
@@ -858,8 +858,8 @@ func (k Keeper) getBeginInfo(
 		return completionTime, height, false, nil
 	}
 	headerInfo := k.HeaderService.HeaderInfo(ctx)
-	unbondingTime, err := k.UnbondingTime(ctx)
-	if err != nil {
+	unbondingTime, err2 := k.UnbondingTime(ctx)
+	if err2 != nil {
 		return completionTime, height, false, err
```

Comment:
```suggestion
		return completionTime, height, false, errors.Join(err, err2)
```

We should error join the two errors, otherwise we miss the second one now.

Reply:
cc @tac0turtle